ml-agents-envs/mlagents_envs/environment.py

```diff
@@ -377,6 +383,18 @@ def get_steps(
         self._assert_behavior_exists(behavior_name)
         return self._env_state[behavior_name]
 
+    def _poll_process(self) -> None:
+        """
+        Check the status of the subprocess. If it has exited, raise a UnityEnvironmentException
+        :return: None
+        """
+        if not self._proc1:
```

Comment:
Side discussion - should we rename `_proc1`? I think it's been called that for a long time, but doesn't really make any sense.

Reply:
For what it is worth my vote is for a rename

Reply:
Renamed to `_process`